src/core.js

```diff
@@ -1316,6 +1317,11 @@ $.extend( $.validator, {
 			return this.optional( element ) || ( value >= param[ 0 ] && value <= param[ 1 ] );
 		},
 
+		// http://jqueryvalidation.org/step-method/
```

Comment:
the docs for this url need to be created by a separate PR in https://github.com/jzaefferer/validation-content